app/src/main/java/org/schulcloud/mobile/controllers/file/BreadcrumbsView.kt

```diff
@@ -70,11 +63,11 @@ open class BreadcrumbsView @JvmOverloads constructor(
         dividerDrawable.setColorFilter(color, PorterDuff.Mode.SRC_ATOP)
     }
 
-    private fun addPartView(path: String, title: String) {
+    private fun addPartView(refOwnerModel: String, owner: String, parent: String?, title: String?) {
         addView(CompatTextView(context).also {
             it.textSize = textSize
```

Comment:
This problem exists for some time, but maybe you can fix it as well. `TextView.setTextSize()` expects a value in sp, but the loaded `textSize` is already converted to pixels. I think it just has to be changed to `setTextSize` with an explicit unit.